go/vt/orchestrator/inst/analysis_dao_test.go

```diff
@@ -424,22 +440,52 @@ func TestGetReplicationAnalysis(t *testing.T) {
 					MysqlHostname: "localhost",
 					MysqlPort:     6708,
 				},
+				DurabilityPolicy:       "none",
 				SourceHost:             "localhost",
 				SourcePort:             6708,
 				LastCheckValid:         1,
 				ReadOnly:               1,
 				SemiSyncReplicaEnabled: 1,
 			}},
 			codeWanted: ReplicaSemiSyncMustNotBeSet,
+		}, {
+			name: "SnapshotKeyspace",
+			info: []*test.InfoForRecoveryAnalysis{{
+				TabletInfo: &topodatapb.Tablet{
+					Alias:         &topodatapb.TabletAlias{Cell: "zon1", Uid: 100},
+					Hostname:      "localhost",
+					Keyspace:      "ks",
+					Shard:         "0",
+					Type:          topodatapb.TabletType_REPLICA,
+					MysqlHostname: "localhost",
+					MysqlPort:     6709,
+				},
+				// Snapshot Keyspace
+				KeyspaceType:     1,
+				DurabilityPolicy: "none",
+				LastCheckValid:   1,
+			}},
+			codeWanted: NoProblem,
+		}, {
+			name: "EmptyDurabilityPolicy",
+			info: []*test.InfoForRecoveryAnalysis{{
+				TabletInfo: &topodatapb.Tablet{
+					Alias:         &topodatapb.TabletAlias{Cell: "zon1", Uid: 100},
+					Hostname:      "localhost",
+					Keyspace:      "ks",
+					Shard:         "0",
+					Type:          topodatapb.TabletType_REPLICA,
+					MysqlHostname: "localhost",
+					MysqlPort:     6709,
+				},
+				LastCheckValid: 1,
+			}},
+			// We will ignore these keyspaces too until the durability policy is set in the topo server
+			codeWanted: NoProblem,
```

Comment:
If the durability policy is not specified in the topo server, VTOrc will ignore the keyspace.

Reply:
hmm.. shouldn't vtorc still watch/repair the keyspace and default the durability policy to "none"?

Reply:
Based on offline discussion, this is fine since
(a) VTOrc is not GA yet
(b) For anyone already using it, there will be a documented upgrade path in the release notes to avoid getting into this situation in the first place.